### ISSUE TITLE
Allow for training without test set

### ISSUE THREAD

**Issue by emanjavacas** [hipster-philology/pandora#85]
Currently, tagger.py wouldn't run unless you provide all three splits. Especially for fine-tunning is highly recommendable to not use the test set, since that would actually lead to overfitting on the test set (since you are getting feedback from your model's performance on the test set). You can of course set the test split to be the same as the dev, then you are running the same evaluation twice, which is wasteful.

**Comment by emanjavacas** [comment]
I just saw there is the `include_test` param in the config file. I guess that's is redundant with the fact of actually providing a path for --test... we could get rid of that config param.

**Comment by Jean-Baptiste-Camps** [comment]
In that case, we should do the same with `include_dev` too, right ?

**Comment by emanjavacas** [comment]
yes, I think so

**Comment by PonteIneptique** [comment]
I think this has been fixed by `setup_from_disk` but I might be wrong
